pkg/common/label.go

```diff
@@ -25,8 +25,8 @@ const (
 	// LabelAnnotationDatasetNum indicates the number of the dataset in specific node
 	LabelAnnotationDatasetNum = LabelAnnotationPrefix + "dataset-num"
 
-	// fluid adminssion webhook inject pod affinity strategy flag
-	LabelFluidSchedulingStrategyFlag = LabelAnnotationPrefix + "enable-scheduling-strategy"
+	// fluid adminssion webhook inject flag
+	EnableFluidInjectionFlag = LabelAnnotationPrefix + "enable-scheduling-strategy"
```

Comment:
Unified to `enable-injection` ? 
```go
	EnableFluidInjectionFlag = LabelAnnotationPrefix + "enable-injection"
```

Reply:
Yes. We'd like use feature gate for different functions. But we need a global flag to enable or disable webhook in general.